packages/feeds-client/src/bindings/react/contexts/StreamFeedContext.tsx

```diff
@@ -1,5 +1,5 @@
 import { createContext, useContext } from 'react';
-import type { Feed } from '../../src/feed';
+import type { Feed } from '@self';
```

Comment:
Might not be the best naming (suggest better) but the paths seemed too long so I decided to introduce the alias here.

Reply:
I hate them too - not sure about the `nx` though - could you try i t out whenever you get the chance, please?

Reply:
Will do !